### ISSUE TITLE
Building while Paused can break paths

### ISSUE THREAD

**Issue by Krutonium** [OpenRCT2/OpenRCT2#13582]
**OS:** Arch Linux
**Version:** 0.3.2-198
**Commit/Build:** 9ef493198

Ghost Queue Lines from a Ride Blueprint can break existing built queue lines if the ghosts are put over them.

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
0. Enable Building while Paused, then Pause the game
1. Have existing queue line in the world
2. Hover a ghost ride's queue line over that path
3. Move it around and see how it breaks the existing path.

**Screenshots / Video:**
![Canada's Wonderland 2020-12-12 16-08-15](https://user-images.githubusercontent.com/3945538/101994982-89d20180-3c94-11eb-98de-d84126088b5b.png)
![Canada's Wonderland 2020-12-12 16-09-42](https://user-images.githubusercontent.com/3945538/101994983-8b032e80-3c94-11eb-9340-67d93f8ce376.png)

**Comment by csunday95** [comment]
I was able to reproduce this, and it seems to also occur when the game isn't paused as well

**Comment by duncanspumpkin** [comment]
Yeah this is a duplicate of the previous issue. Pause doesn't matter.
